app/src/main/java/org/p2p/wallet/home/ui/main/bottomsheet/ActionsBottomSheet.kt

```diff
@@ -53,6 +58,16 @@ class HomeActionsBottomSheet : BottomSheetDialogFragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         with(binding) {
+            if (moonpaySellRepository.isSellAllowedForUser()) {
```

Comment:
that's not good. 
Let's move this check to presenter and just override a separate fun showing the `viewAction` view

Reply:
extracted to presenter